pkg/api/conversion.go

```diff
@@ -53,12 +49,12 @@ func init() {
 			out.UID = in.UUID
 
 			if in.ID != "" {
-				out.SelfLink = "/api/v1beta1/boundPods/" + in.ID
```

Comment:
I though that it's an apiserver endpoint, but still - I didn't found /boundPods handler in apiserver.
Should I revert this change?